z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-control-plane/tasks/main.yaml

```diff
@@ -205,5 +216,5 @@
     PYTHONWARNINGS: 'ignore::UserWarning'
   when:
   - disk_type == "scsi"
-  - vm_type == "zvm"
```

Comment:
where is the entry of kvm BFV? 
`when disk_type=='scsi' & volume_type_id is defined ` ?